LSP.sublime-settings

```diff
@@ -97,19 +124,49 @@
       "languageId": "java"
     },
   },
+
+  // Show activity messages in the status bar for a few seconds.
+  // Example: Starting pyls ...
   "show_status_messages": true,
+
+  // Show permanent language server status in the status bar.
   "show_view_status": true,
+
+  // Open and close the diagnostics panel automatically,
+  // depending on available diagnostics.
   "auto_show_diagnostics_panel": true,
+
+  // Show in-line diagnostics using phantoms for unchanged files.
   "show_diagnostics_phantoms": false,
+
+  // Show diagnostics description in status bar
+  // if the cursor hovers complained code.
```

Comment:
Should be something like "when the cursor is on code with diagnostics"

Reply:
I am not too happy with it, too. Wanted to avoid duplicated usage of the word diagnostics.

Maybe:

```
  // Show the diagnostics description of the code
  // under the cursor in status bar if available.
```
or so?

Reply:
:+1: